### ISSUE TITLE
Use conda's distribution of RabbitMQ for arm64 architecture, when available

### ISSUE THREAD

**Issue by yakutovicha** [aiidateam/aiida-prerequisites#57]
Currently, arm64 architecture is not supported for conda builds of RabbitMQ, see [here](https://anaconda.org/conda-forge/rabbitmq-server). As a temporary solution, we used the package available in the system for arm64. However, as soon as conda builds for arm architecture are available, we should use them instead.

I asked a [question in RMQ google groups](https://groups.google.com/g/rabbitmq-users/c/t2UGIJpaz3M). Let's see what and when they reply.

**Comment by ltalirz** [comment]
Just wondering: if the ubuntu packages for rabbitmq are more up to date in terms of coverage of different CPU architectures, what is the reasoning for switching to conda for managing rabbitmq (and postgresql)?

Have you compared the size of installs via conda environment vs the size of install via ubuntu package?

**Comment by yakutovicha** [comment]
> Just wondering: if the ubuntu packages for rabbitmq are more up to date in terms of coverage of different CPU architectures, what is the reasoning for switching to conda for managing rabbitmq (and PostgreSQL)?

The flexibility and easiness of maintenance, I would say. Conda environment lets us be decoupled from the version that is bound to the Ubuntu version.

The `arm` support will come at some point, I am not worried about it. As a temporary solution we have this:

https://github.com/aiidateam/aiida-prerequisites/blob/5a1f540d794e8ddbac52e3ddae2beaa57250eac5/Dockerfile#L103-L114


> Have you compared the size of installs via conda environment vs the size of install via ubuntu package?

yes, the comparison can be found [here](https://github.com/aiidateam/aiida-prerequisites/issues/35#issuecomment-1199885318)

**Comment by ltalirz** [comment]
> Conda environment lets us be decoupled from the version that is bound to the Ubuntu version.

Is the version really fixed by Ubuntu?

The standard [postgresql installation instructions](https://www.postgresql.org/download/linux/ubuntu/) explain how to install the latest postgres version (or a specific one).

For rabbitmq you can find the `.deb` file for each release on github https://github.com/rabbitmq/rabbitmq-server/releases that I guess you can install with `sudo dpkg -i package.deb` (but one will need to install dependencies via apt). That does look a bit more problematic...

**Comment by yakutovicha** [comment]
> Is the version really fixed by Ubuntu?

Ok, let me rephrase it: not fixed, but available in Ubuntu without further changes.

> The standard [postgresql installation instructions](https://www.postgresql.org/download/linux/ubuntu/) explain how to install the latest postgres version (or a specific one).
> 
> For rabbitmq you can find the `.deb` file for each release on github https://github.com/rabbitmq/rabbitmq-server/releases that I guess you can install with `sudo dpkg -i package.deb` (but one will need to install dependencies via apt). That does look a bit more problematic...

Feel free to give it a try.
